### PR TITLE
add react-native-web example

### PR DESCRIPTION
**Pull request by Gregor1971**
adds a project to the examples dir:
a basic working styleguide for a react-native app using react-native-web
the app was created via crna

... as promised in #675

### CODE REVIEW

File: examples/react-native-web/styleguide.config.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+const path = require('path');
+
+module.exports = {
+	verbose: false,
```
Comment (sapegin) [inline examples/react-native-web/styleguide.config.js]: verbose, components and ignore options are the same as defaults, you don’t need them here.

File: examples/react-native-web/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "webpack": "^3.10.0"
+  },
+  "main": "./node_modules/react-native-scripts/build/bin/crna-entry.js",
+  "scripts": {
+    "start": "react-native-scripts start",
+    "eject": "react-native-scripts eject",
+    "android": "react-native-scripts android",
+    "ios": "react-native-scripts ios",
+    "styleguide": "styleguidist server",
+    "test": "node node_modules/jest/bin/jest.js --watch"
```
Comment (sapegin) [inline examples/react-native-web/package.json]: Was this command generated? Just `jest --watch` is enough.

Comment (Gregor1971) [reply examples/react-native-web/package.json]: yes, crna generates the script entries.  best to leave it as is so it matches what users see in theirs.

File: examples/react-native-web/styleguide.config.js (hunk 1 above)
Comment (Gregor1971) [reply examples/react-native-web/styleguide.config.js]: updates pushed.


I had stared with my much more complex config file and pared it down to this... not enough, though

**Comment by sapegin** [comment]
Now there are several ESLint warnings.

**Comment by sapegin** [comment]
And remove `yarn.lock` please.

**Comment by Gregor1971** [comment]
@sapegin ,
This pr is just a simple example.  If linting and code coverage for examples are that important, you can just close this pr.

**Comment by sapegin** [comment]
There's no point in making exceptions for examples: they are code that someone will have to maintain too. Same simple rules apply to all code in the project.

**Comment by Gregor1971** [comment]
@sapegin 
look at the coverage error.   it's a file I didn't touch.   pr closed.  best wishes

**Pull request closed without merging**